pkg/http-server/webhook-scan_test.go

```diff
@@ -0,0 +1,248 @@
+package httpserver
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/accurics/terrascan/pkg/k8s/dblogs"
+	"github.com/gorilla/mux"
+	v1 "k8s.io/api/admission/v1"
+)
+
+func TestUWebhooks(t *testing.T) {
+	testFilePath := "./k8s_testdata/testconfig.json"
```

Comment:
we should form all paths using `filepath.Join()`

Reply:
Done